src/components/RSVPControl.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Link } from 'react-router'
+const { object, func } = React.PropTypes
+import { filter } from 'lodash'
+import Avatar from './Avatar'
+import Dropdown from './Dropdown'
+import { changeEventResponse } from '../actions'
+
+export default class RSVPControl extends React.Component {
+  static propTypes = {
+    currentUser: object,
+    post: object,
+    dispatch: func
+  }
+
+  render () {
+    var props = this.props
+    let { post, currentUser } = props
```

Comment:
you only need props for dispatch, so i would do:

```js
let { post, currentUser, dispatch } = this.props
// ...
var eventResponseClicked = response => e => dispatch(...)
```

btw, you can also skip the function nesting and just do something like this:
```js
var pickResponse = choice => dispatch(...)

<button onClick={() => pickResponse('yes')}>yes</button>
```